src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * Add method `getConstraint()` to `ConstraintViolationInterface`
  * Add `Uuid::TIME_BASED_VERSIONS` to match that a UUID being validated embeds a timestamp
  * Add the `pattern` parameter in violations of the `Regex` constraint
+ * Add a `NotSuspicious` constraint to validate a string is not a spoof attempt
```

Comment:
About the name, I'd propose `NoSuspiciousCharacters`, to make it clear that this is not about the full value being suspicious, but only about some characters being

Reply:
Hm currently the full value is suspicious if any of its character is. ICU says it checks strings, not characters, so I’m not sure we have to make the distinction.

> Checking whether an individual **string** is likely to be an attempt at confusing the reader
>
> https://unicode-org.github.io/icu-docs/apidoc/dev/icu4j/com/ibm/icu/text/SpoofChecker.html

Reply:
yeah, but it also says "is likely to be an attempt at confusing the reader", not "is suspicious". A string might be suspicious for other reasons than failing the ICU checks.

Reply:
Okay I understand. In fact the term `suspicious` only comes from PHP’s `Spoofchecker` so I guess we could even not using it.

Going  for `NoSuspiciousCharacters` for now.